magento/magento2
# Added processing of missed 'Fields enclosure' on admin exports

### Description

The value of "Fields Enclosure" checkbox on admin export page (System -> Export) is not passed to appropriate export model, therefore enclosure behavior is not handled.
This Pull Request is fixing that.

### Manual testing scenarios

1. Ensure you have some products and custom attributes.
2. Export products through backend with the parameters:
- Entity Type: Products
- Export File Format: CSV
- Fields Enclosure: Yes
3. Download and open generated CSV file.

All attribute values in `additional_attributes` column must be wrapped by double quotes `"`, but they aren't.

### Related Issues

Probably, it's worth to mention here closed issue magento/magento2#14642, there was the similar problem with `skip_attr` field.



### Resolved issues:
1. [x] resolves magento/magento2#29554: Added processing of missed 'Fields enclosure' on admin exports
